roles/k3s/tasks/config-k3s-service.yml

```diff
@@ -60,3 +60,9 @@
     daemon_reload: yes
     state: started
     enabled: yes
+
+- name: Install Kubernetes Python client
+  become: true
+  pip:
+    name: kubernetes
+    state: present
```

Comment:
Did you verify that this installed client into `/etc/ansible/venv`, such as by deleting the venv, then re-creating with bootstrap-servers prior to running this playbook?

Also, we should pin the version of python-kubernetes installed, or we might get unexpected breakage later

Reply:
sure looking into